docs/developer-docs/latest/update-migration-guides/migration-guides/v4/code/backend/graphql.md

```diff
@@ -149,7 +149,7 @@ module.exports = {
 
 ::: note NOTES
 
-* In Strapi v4, REST controllers and GraphQL resolvers are not coupled anymore. The business logic is implemented in services and called either from the controller or the resolver. This approach keeps the business logic in one place so both REST and GraphQL can be customized the way you want.
+- In Strapi v4, REST controllers and GraphQL resolvers are not coupled any more. The business logic is implemented in services and called either from the controller or the resolver. This approach keeps the business logic in one place so both REST and GraphQL can be customized the way you want.
```

Comment:
This shouldn't be changed. It won't pass our linter (and won't be consistent with the lines below).